src/test/java/org/folio/rest/api/IllMigrationScriptTest.java

```diff
@@ -0,0 +1,43 @@
+package org.folio.rest.api;
+
+import static org.hamcrest.CoreMatchers.is;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import io.vertx.core.json.JsonObject;
+
+public class IllMigrationScriptTest extends MigrationTestBase {
+  private static final String MIGRATION_SCRIPT = loadScript("updateIllPolicyWillNotLend.sql");
+  private static final String WILL_NOT_LEND_ID = "b0f97013-87f5-4bab-87f2-ac4a5191b489";
+  private static final String WILL_NOT_LEND_UPDATED_NAME = "Will not lend";
+
+  @Test
+  public void canMigrateIllPolicies() throws Exception {
+    Map<String, JsonObject> initialIllPolicies = new HashMap<String, JsonObject>();
+    illPoliciesClient.getAll().forEach(policy -> {
+      initialIllPolicies.put(policy.getString("id"), policy);
```

Comment:
I think, the policy should already have a proper name because it is being loaded as ref data.

I'd suggest to manually update it with the old name before executing the migration script.

Reply:
I have confirmed that the tests still pass when starting from '[w]ill not lend' in the reference data.